src/workbooks/governance/workbook.json

```
@@ -2035,139 +1798,444 @@
+                              {
+                                "columnId": "microsoft.storage/storageaccounts/queueservices-Capacity-QueueCapacity",
+                                "label": "Queue capacity"
+                              },
+                              {
+                                "columnId": "microsoft.storage/storageaccounts/queueservices-Capacity-QueueCapacity Timeline",
+                                "label": "Queue capacity Timeline"
+                              },
+                              {
+                                "columnId": "microsoft.storage/storageaccounts/tableservices-Capacity-TableCapacity",
+                                "label": "Table capacity"
+                              },
+                              {
+                                "columnId": "microsoft.storage/storageaccounts/tableservices-Capacity-TableCapacity Timeline",
+                                "label": "Table capacity Timeline"
+                              }
+                            ]
                           },
-                          "tileSettings": {
-                            "titleContent": {
-                              "columnMatch": "type",
-                              "formatter": 1
+                          "sortBy": [
+                            {
+                              "itemKey": "$gen_link_$gen_group_0",
+                              "sortOrder": 1
+                            }
+                          ],
+                          "showExportToExcel": true
+                        },
+                        "conditionalVisibility": {
+                          "parameterName": "selectedTab",
+                          "comparison": "isEqualTo",
+                          "value": "Capacity"
+                        },
+                        "showPin": true,
+                        "name": "storage account capacity metrics",
+                        "styleSettings": {
+                          "margin": "0 10px 0 10px"
+                        }
+                      }
+                    ]
+                  },
+                  "conditionalVisibility": {
+                    "parameterName": "Subscription",
+                    "comparison": "isNotEqualTo",
+                    "value": ""
+                  },
+                  "name": "Storage account & backup"
+                },
+                {
+                  "type": 12,
+                  "content": {
+                    "version": "NotebookGroup/1.0",
+                    "groupType": "editable",
+                    "items": [
+                      {
+                        "type": 1,
+                        "content": {
+                          "json": "Azure Backup now provides a set of customizable reporting templates to help you generate audit evidence reports for backup in an easier way. [Click here](https://aka.ms/BCDRAuditReportTemplates) to access these templates.",
```

nit: "Click here" links are generally an antipattern for accessibility. It's usually best to describe the thing you're linking to, but "learn more" is also a common approach in Microsoft, so that may be better. Your call.
```suggestion
                          "json": "Azure Backup now provides a set of customizable reporting templates to help you generate audit evidence reports for backup in an easier way. [Learn more](https://aka.ms/BCDRAuditReportTemplates).",
```